packages/core/contracts-ovm/insured-bridge/implementation/OptimismMessenger.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
this is super clean, I really like this pattern.

I dont think that the Arbitrum one will be as clean though...

Reply:
Arbitrum one will require additional params like `gasPrice`, so I was thinking that either we create a `MessageParams` struct or just add more params to `relayMessage` that we can throwaway in the Optimism case. Luckily, the optimism params are a strict subset of the arbitrum ones...